### ISSUE TITLE
Copy screenshot to clipboard

### ISSUE THREAD

**Issue by thany** [kufii/CodeSnap#45]
When I hit the aperture, it prompts me to save the screenshot to disk. That's a tedious way to move a screenshot over to a program that supports pasting in an image from the clipboard.

Can you please copy the screenshot itself to the clipboard when the aperture button is pressed? And make it an option whether or not to prompt to save to disk? I for one, never save something to disk, because all programs I use to share any screenshot, support pasting an image from the clipboard.

In short, I want to:
1. Activate CodeSnap
2. Select code
3. Press the aperture button
4. Paste into, for example, a Slack message

That would be litterally all there is to it, ideally.

**Comment by djD-REK** [comment]
+1

This would also be useful for my primary use case (writing articles in a
WYSIWYG editor).

And lots of tools I use (emoji tools) overwrite the clipboard whether or
not I want them to, so I think there's a precedent.

**Comment by thany** [comment]
> And lots of tools I use (emoji tools) overwrite the clipboard whether or
not I want them to, so I think there's a precedent.

Exactly, overwriting the clipboard isn't a bad thing, as far as I'm concerned. The clipboard should be considered volatile anyway.
But if worse comes to worse, it could be an opt-out option.

**Comment by AndhikaWB** [comment]
If I remember correctly, clicking the preview image and pressing CTRL + C will make the preview image blink for a second and also save it to clipboard. Unfortunately, I can't confirm it again right now since CodeSnap is currently broken on latest VSCode update.

**Comment by kufii** [comment]
pressing ctrl/cmd+C does indeed copy to clipboard. In a later UX improvement I will make this clearer
